### ISSUE TITLE
Trying to publish a room which is blocked by the server doesn't do anything

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#9467]
Synapse can return 403 forbidden for the request, however the toggle swallows errors. We should at least give a dialog like we do for room aliases.

**Comment by t3chguy** [comment]
Duplicate of https://github.com/vector-im/element-web/issues/8632
